src/appComponents/sponsors/SponsorsController.js

```diff
@@ -19,7 +19,7 @@ sponsorsModule.controller('SponsorsController',
         var sc = this;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unexpected `this`.](https://www.codacy.com/app/triplez/open-event-webapp/file/2418082362/issues/source?bid=3233857&fileBranchId=3237740#l19)